dataplanes/vpp/README.md

```diff
@@ -0,0 +1,6 @@
+nsmvpdataplane
+==============
+
+## Network service mesh Dataplane module based on VPP
+
+**State: WIP**
```

Comment:
Quick note: Can you add a comment here indicating that once we hit the initial milestone of the demo at KubeCon we'll move this into a separate repository? We may also want a note that that we're putting this here as we iterate on the data plane API to make progress towards the demo, and that we will not accept other data planes inside the main tree.

Reply:
Done.